src/Directory.Build.targets

```diff
@@ -17,8 +17,8 @@
     <PackageReference Update="Uno.SourceGeneration" Version="1.32.0-dev.297" />
     <PackageReference Update="Uno.Core" Version="1.29.0-dev.93" />
     <PackageReference Update="Uno.Core.Build" Version="1.29.0-dev.93" />
-    <PackageReference Update="Uno.Wasm.Bootstrap" Version="1.0.3" />
-    <DotNetCliToolReference Update="Uno.Wasm.Bootstrap.Cli" Version="1.0.3" />
+    <PackageReference Update="Uno.Wasm.Bootstrap" Version="1.0.4" />
```

Comment:
`1.0.4` or `1.0.5`? You should adjust the title of the PR ;-)

Reply:
Really was 1.0.5, mixed up the version numbers :)